src/components/ha-control-slider.ts

```diff
@@ -298,6 +382,87 @@ export class HaControlSlider extends LitElement {
         width: var(--control-slider-thickness);
         height: 100%;
       }
+      .container {
+        position: relative;
+        height: 100%;
+        width: 100%;
+        --handle-size: 4px;
+        --handle-margin: calc(var(--control-slider-thickness) / 8);
+      }
+      .tooltip {
+        user-select: none;
+        position: absolute;
+        background-color: var(--clear-background-color);
+        color: var(--primary-text-color);
+        font-size: var(--control-slider-tooltip-font-size);
+        border-radius: 0.8em;
+        padding: 0.2em 0.4em;
+        opacity: 0;
+        white-space: nowrap;
+        box-shadow: 0 2px 5px rgba(0, 0, 0, 0.2);
+        transition:
+          opacity 180ms ease-in-out,
+          left 180ms ease-in-out,
+          bottom 180ms ease-in-out;
+        --tooltip-margin: -4px;
+        --tooltip-range: 100%;
+        --tooltip-offset: 0px;
+        --handle-spacing: calc(2 * var(--handle-margin) + var(--handle-size));
+        --tooltip-position: calc(
+          min(
+            max(
+              var(--value) * var(--tooltip-range) + var(--tooltip-offset),
+              0%
+            ),
+            100%
+          )
+        );
+      }
+      .tooltip.start {
+        --tooltip-offset: calc(-0.5 * (var(--handle-spacing)));
```

Comment:
We should give these more distinctive names so they will not clash with other tooltips in other components